.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+2.6.8
```

Comment:
It's recommended to use the oldest supported Ruby version here, so that `rubocop` can analyze    the files accordingly 👀

Reply:
Running rubocop against ruby2.5 syntax with this change https://github.com/airbnb/sparsam/pull/33/files#diff-4f894049af3375c2bd4e608f546f8d4a0eed95464efcdea850993200db9fef5cR2

To be honest, we should probably drop support for 2.5 and 2.6 soon, they're past EOL (2.5 is _well past_ EOL).